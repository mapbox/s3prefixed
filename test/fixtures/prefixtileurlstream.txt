{prefix}/hi/13/125/107
{prefix}/hi/13/125/108
{prefix}/hi/13/125/109
{prefix}/hi/13/125/110
{prefix}/hi/13/126/105
{prefix}/hi/13/126/106
{prefix}/hi/13/126/107
{prefix}/hi/13/126/108
{prefix}/hi/13/126/109
{prefix}/hi/13/126/110
{prefix}/hi/13/127/105
{prefix}/hi/13/127/106
{prefix}/hi/13/127/107
{prefix}/hi/13/127/108
{prefix}/hi/13/127/109
{prefix}/hi/13/127/110
{prefix}/hi/13/128/105
{prefix}/hi/13/128/106
{prefix}/hi/13/128/107
{prefix}/hi/13/128/108
{prefix}/hi/13/128/109
{prefix}/hi/13/128/110
{prefix}/hi/13/129/105
{prefix}/hi/13/129/106
{prefix}/hi/13/129/107
{prefix}/hi/13/129/108
{prefix}/hi/13/129/109
{prefix}/hi/13/129/110
{prefix}/hi/13/130/105
{prefix}/hi/13/130/106
{prefix}/hi/13/130/107
{prefix}/hi/13/130/108
{prefix}/hi/13/130/109
{prefix}/hi/13/125/106
{prefix}/hi/13/125/105
{prefix}/hi/13/130/110
