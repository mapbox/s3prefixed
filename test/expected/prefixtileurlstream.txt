db/hi/13/125/107
dc/hi/13/125/108
dd/hi/13/125/109
de/hi/13/125/110
e9/hi/13/126/105
ea/hi/13/126/106
eb/hi/13/126/107
ec/hi/13/126/108
ed/hi/13/126/109
ee/hi/13/126/110
f9/hi/13/127/105
fa/hi/13/127/106
fb/hi/13/127/107
fc/hi/13/127/108
fd/hi/13/127/109
fe/hi/13/127/110
09/hi/13/128/105
0a/hi/13/128/106
0b/hi/13/128/107
0c/hi/13/128/108
0d/hi/13/128/109
0e/hi/13/128/110
19/hi/13/129/105
1a/hi/13/129/106
1b/hi/13/129/107
1c/hi/13/129/108
1d/hi/13/129/109
1e/hi/13/129/110
29/hi/13/130/105
2a/hi/13/130/106
2b/hi/13/130/107
2c/hi/13/130/108
2d/hi/13/130/109
da/hi/13/125/106
d9/hi/13/125/105
2e/hi/13/130/110
